watermark/watermark.py

```diff
@@ -233,7 +233,9 @@ def _print_all_import_versions(vars):
             if isinstance(val, types.ModuleType):
                 if val.__name__ != 'builtins':
                     try:
-                        to_print.add((val.__name__, val.__version__))
+                        for v in ["VERSION", "__version__"]:
+                            if hasattr(val, v):
+                                to_print.add((val.__name__, getattr(val, v)))
```

Comment:
I wonder if we should add a break here. I.e.,


    if hasattr(val, v):
        to_print.add((val.__name__, getattr(val, v)))
        break

just in case some package has both `VERSION` and `__version__` attributes.